test/Microsoft.Health.Fhir.Tests.E2E/Rest/VReadTests.cs

```diff
@@ -30,7 +31,7 @@ public VReadTests(HttpIntegrationTestFixture<Startup> fixture)
         [Trait(Traits.Priority, Priority.One)]
         public async Task WhenGettingAResource_GivenAnIdAndVersionId_TheServerShouldReturnTheAppropriateResourceSuccessfully()
         {
-            Observation createdResource = await Client.CreateAsync(Samples.GetDefaultObservation());
+            Observation createdResource = await Client.CreateAsync(Samples.GetDefaultObservation().ToPoco<Observation>());
```

Comment:
What's the plan for when we need to start testing STU3 and R4?

Reply:
This is a separate task. We'll need to leverage the new testing attributes. We'll also need to spike perhaps wrapping the FhirClient with an agnostic proxy